HerPublicWebsite/Program.cs

```diff
@@ -61,6 +61,14 @@ public static void Main(string[] args)
                     "Send policy team update email",
                     rjs => rjs.SendPolicyTeamUpdate(),
                     "0 7 * * 1");
+            
+            app
+                .Services
+                .GetService<IRecurringJobManager>()
+                .AddOrUpdate<PendingReferralNotificationService>(
+                    "Send monthly pending referral report",
+                    rjs => rjs.SendPendingReferralNotifications(),
+                    "0 0 1 * *");

```

Comment:
For consistency with the `PolicyTeamUpdateService` one, let's run this in the early morning rather than at midnight.

That one runs at 07:00. Just to be overcautious, let's add this new one at 07:15.